cves/kernel/CVE-2021-37576.yml

```diff
@@ -75,7 +76,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: ['https://bugzilla.redhat.com/show_bug.cgi?id=1986506']
```

Comment:
I think this is specifically looking for bug IDs from https://bugzilla.kernel.org/, so I don't think this would qualify.